src/azure-cli/azure/cli/command_modules/appservice/custom.py

```diff
@@ -2314,6 +2328,420 @@ def remove_triggered_webjob(cmd, resource_group_name, name, webjob_name, slot=No
     return client.web_apps.delete_triggered_web_job(resource_group_name, name, webjob_name)
 
 
+def webapp_list_hc(cmd, name, resource_group):
```

Comment:
are we not supporting these commands for slots? since the feature is supported on slots we should add slot support as well